src/content/community/support/index.md

```diff
@@ -38,8 +38,11 @@ Looking for an Ethereum wallet? [Explore our full list of Ethereum wallets](/wal
 
 Building can be hard. Here are some development focused spaces with experienced Ethereum developers that are happy to help.
 
+- [Alchemy University](https://university.alchemy.com/#starter_code)
+- [Chainlink](https://blog.chain.link/learn-blockchain-full-stack-web3-javascript-smart-contract-development/)
 - [CryptoDevs discord](https://discord.gg/Z9TA39m8Yu)
 - [Ethereum StackExchange](https://ethereum.stackexchange.com/)
+- [Nader Dabit](https://naderdabit.notion.site/Nader-s-web3-Resources-for-Developers)
```

Comment:
```suggestion
```
I cant access this, seems to be restricted

Reply:
Would developer dao maybe be more appropriate? I know Nader is quite involved with that community

Reply:
Thanks for flagging Corwin! Maybe with his recent move to Aave / Lens Protocol, some view settings changed? Does this work at all - https://naderdabit.notion.site/Nader-s-web3-Resources-for-Developers-a200ed2ef21c4d578dc158df2b882c63? I'm seeing it on my end in incognito too. 

If not, sure developer dao could be good. Maybe this one that looks more general? https://blog.developerdao.com/free-resources-to-learn-solidity-ethereum-blockchain-fullstack-dapps-and-smart-contract.